lib/forgery.ex

```diff
@@ -94,7 +94,11 @@ defmodule Forgery do
       def make(factory_name, fields \\ %{})
 
       def make_many(factory_name, amount, fields \\ %{}) when is_integer(amount) do
```

Comment:
should we add a guard that amount should be >= 0?

Reply:
Sure, but what should be the result when the amount is < 0? I mean for amount = 0 I think it's correct return an empty list

Reply:
maybe let's do:

```ex
def make_many(factory_name, amount, fields \\ %{})

def make_many(_factory, 0, _fields), do: []
def make_many(factory_name, amount, fields) when amount > 0 do
  ...
end
```